tests/installation/disk_activation_iscsi.pm

```diff
@@ -0,0 +1,72 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: initial installation: iSCSI client for the supportserver
+#          supportserver to provide one iSCSI target which will 
+#          provide a multipathed second disk to the client.
+# Maintainer: Klaus Wagner <kgw@suse.com>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $self = shift;
+    my $iscsi_iqn = "iqn.2016-02.de.openqa";
+    my $iscsi_server_ip = "10.0.2.1";
```

Comment:
If you use test suite variables with default values here, you will be able to change iscsi parameters to use any available iscsi server and possibly without `SUPPORT_SERVER`.

Reply:
Good point. OTOH most of the needles currently associated with this module are based on this special case, so I'd have to rework them, too.

Reply:
Done in amended PR commit e150dcc (below). Also amended the associated needle PRs. Turned out only one needle needed a significant change.